.travis.yml

```diff
@@ -6,9 +6,9 @@ sudo: false
 cache:
   directories: [$HOME/.cache/pip]
 
-install: pip install sphinx~=1.3.0 git+https://github.com/fabpot/sphinx-php.git
+install: pip install -r _build/.requirements.txt sphinx==1.3.0
```

Comment:
Thanks! I've turned this comment into a proper issue (#7422) to not lose track of it.